.github/workflows/assign-to-one-project.yml

```
@@ -0,0 +1,20 @@
+name: Assign to one project
```

```suggestion
name: frontend-platform-project-inbox
```

I think that will be better in the actual CI runs.